_episodes/07-branches.md

```diff
@@ -80,13 +101,13 @@ $ git branch
 ~~~
 {: .output}
 
+Before we used the `checkout` command to checkout a commit using
+commit hashs or `HEAD`. This is similar but now we have given our most
```

Comment:
:laughing: yeah reading that back even I was like "what was I trying to say there?"

I think I was grasping for a comparison to the other ways in which we were using `git checkout`. Kind of to generalize the purpose of that command. That's not commonly how I think of checking out a branch either. I generally think of it as a change of purpose for what I'm working on in the repository. This part is tricky...